src/nikita/Assignment23UpperLower.java

```diff
@@ -0,0 +1,31 @@
+package nikita;
+
+public class Assignment23UpperLower {
+
+	int ucount,lcount,dcount,scount;
+
+	void getUppercase(String input) {
+		for(int i=0; i<input.length();i++) {
+			if(Character.isUpperCase(input.charAt(i))){
```

Comment:
you can write char ch = input.charAt(index) outside of conditional statement, so that you no need to rewrite again and again. you just need to write like Character.isUpperCase(ch)